bats_test/network_tools.md

```diff
@@ -0,0 +1,35 @@
+# Network tests
+
+Useful to monitor both inbound and outbound traffic to BATS Docker
+
+Disclaimer: *Might now work on MacOS*
```

Comment:
Why so? :/

Reply:
Untested. 
Networking on Mac is a bit limited so I cannot assure everything works there. 
https://docs.docker.com/desktop/mac/networking/